lib/travis/api/app.rb

```diff
@@ -195,6 +195,10 @@ def self.setup_travis
         if use_monitoring? and not console?
           setup_monitoring
         end
+
+        def Redis.instance
```

Comment:
I like this idea, but you have placed it within a method. Maybe it should be its own module, so then the API would look like `Travis::Redis.instance`.